plugins/autoloader/prism-autoloader.js

```diff
@@ -253,42 +253,55 @@
 	}
 
 	/**
-	 * Returns the path to a grammar, using the language_path and use_minified config keys.
+	 * Returns all additional dependencies of the given element defined by the `data-dependencies` attribute.
 	 *
-	 * @param {string} lang
-	 * @returns {string}
+	 * @param {Element} element
+	 * @returns {string[]}
 	 */
-	function getLanguagePath(lang) {
-		return config.languages_path +
-			'prism-' + lang
-			+ (config.use_minified ? '.min' : '') + '.js'
+	function getDependencies(element) {
+		var deps = (element.getAttribute('data-dependencies') || '').trim();
+		if (!deps) {
+			var parent = element.parentElement;
+			if (parent && parent.tagName.toLowerCase() === 'pre') {
+				deps = (parent.getAttribute('data-dependencies') || '').trim();
+			}
+		}
+		return deps ? deps.split(/\s*,\s*/g) : [];
```

Comment:
We have a couple APIs that look up the tree for attributes like this. Is it worthwhile to make this "inherited" like that?

Reply:
What do you mean by "look up the tree"? It only checks the `code` and the `pre` elements. We have a few that look for classes until they reach the root of the documents but this isn't one of them.

Reply:
Yeah, I was wondering if it would be worth doing that, but if not, no worries.